app/views/accounts/contacts/_form.html.erb

```diff
@@ -36,25 +37,24 @@
       <% end %>
     </select>
   </div> -->
-
   <div class="h-0.5 w-full bg-light-palette-p3"></div>
-  <h2 class= "typography-body-s-lh150 text-dark-gray-palette-p1">Marcador de conversas do chatwoot</h2>
+  <h2 class= "typography-body-s-lh150 text-dark-gray-palette-p1">
+    <%= t('activerecord.attributes.contact.chatwoot_conversations_label_list') %>
+  </h2>
   <% if contact.chatwoot_conversations_labels.empty? %>
-    <p class="typography-text-m-lh150 text-dark-gray-palette-p1">Não há marcador de conversas</p>
+    <p class="typography-text-m-lh150 text-dark-gray-palette-p1">
+      <%= t('views.accounts.contacts.form.chatwoot_conversations_label_list_empty') %>
+    </p>
   <% else %>
     <%= render 'components/list_labels', labels: contact.chatwoot_conversations_label_list, max_width: 'none', justify_content: 'start' %>
   <% end %>
   <div class="h-0.5 w-full bg-light-palette-p3"></div>
   <%= render 'accounts/custom_attributes', m: contact, f: form %>
   <div class="flex gap-2 self-end">
     <%= link_to account_custom_attributes_definitions_path(current_user.account), class:"btn-secondary flex items-center gap-2" do %>
-    <i data-lucide="plus"></i>
-    Adicionar campo personalizado
+      <i data-lucide="plus"></i>
+      <%= t('activerecord.models.custom_attribute_definition.add') %>
     <% end %>
     <%= form.submit class: 'btn-primary cursor-pointer' %>
   </div>
 <% end %>
-
-
-
```

Comment:
This change removes blank lines. This is good for improving code organization and consistency.